core/src/test/java/io/questdb/test/jit/CompiledFilterIRSerializerTest.java

```diff
@@ -847,9 +847,16 @@ public void testInSubSelect() throws Exception {
         serialize("asymbol in (select asymbol from tab limit 1)");
     }
 
-    @Test(expected = SqlException.class)
+    @Test
     public void testTimestampInLiteral() throws Exception {
```

Comment:
Please add a test for `atimestamp in null`

Reply:
What's the expected output of such case? Evaluate to `false` or throw a `SqlException`?

Reply:
It should be same as in `IntervalUtils#parseAndApplyIntervalEx()` with null `seq` input. In fact, you could use that method to make sure that the behavior is the same as in `InTimestampStrFunctionFactory`. To do that, you'd need to parse null literal (`tok == null || SqlKeywords.isNullKeyword(tok)`).

Reply:
Added null handling leveraging `IntervalUtils.parseAndApplyIntervalEx` and unit test for it